### PR TITLE
feat(list-editor): insert a child node at a specific position (DSP-1301)

### PR DESCRIPTION
**Pull request by mdelez**
resolves DSP-1301

### CODE REVIEW

**Comment by mdelez** [comment]
tests won't run on GithubCi until a version of DSP-JS is released with this change https://github.com/dasch-swiss/dsp-js-lib/pull/322

**Comment by kilchenmann** [comment]
> tests won't run on GithubCi until a version of DSP-JS is released with this change [dasch-swiss/dsp-js-lib#322](https://github.com/dasch-swiss/dsp-js-lib/pull/322)

I can do a release of js-lib. The problem is the following: PR #322 is marked as `refactor` which is ignored by release-please (s. [contribution documentation](https://docs.dasch.swiss/developers/dsp/contribution/#type) --> "The first three items on this list (fix, feat and type with `!`) are taken into account for the release notes and have an effect on the version number.") and this is why the github actions didn't create a release PR. We could create a new release manually or waiting for the next merged PR.

**Comment by kilchenmann** [comment]
@mdelez p.s. I created a user story about the `refactor`-topic in PRs: DSP-1378

**Comment by mdelez** [comment]
> > tests won't run on GithubCi until a version of DSP-JS is released with this change [dasch-swiss/dsp-js-lib#322](https://github.com/dasch-swiss/dsp-js-lib/pull/322)
> 
> I can do a release of js-lib. The problem is the following: PR #322 is marked as `refactor` which is ignored by release-please (s. [contribution documentation](https://docs.dasch.swiss/developers/dsp/contribution/#type) --> "The first three items on this list (fix, feat and type with `!`) are taken into account for the release notes and have an effect on the version number.") and this is why the github actions didn't create a release PR. We could create a new release manually or waiting for the next merged PR.

oh cool, thanks for the info! It's not urgent as I can use yalc for the time being until another PR is merged :)

**Comment by mdelez** [comment]
Also waiting for this change from DSP-JS https://github.com/dasch-swiss/dsp-js-lib/pull/324

**Review by flavens — APPROVED** [review]

File: src/app/project/list/list-item-form/edit-list-item/edit-list-item.component.ts
Comment (kilchenmann) [inline src/app/project/list/list-item-form/edit-list-item/edit-list-item.component.ts:2]: This long line should be aligned

Comment (mdelez) [reply src/app/project/list/list-item-form/edit-list-item/edit-list-item.component.ts]: done in 0b490f3

**Review by kilchenmann — APPROVED** [review]

**Pull request merged**
